lib/elixir_search_extractor/search_keyword/queries/keyword_query.ex

```diff
@@ -0,0 +1,13 @@
+defmodule ElixirSearchExtractor.SearchKeyword.Queries.KeywordQuery do
+  import Ecto.Query
+  alias ElixirSearchExtractor.FileUpload.Schemas.KeywordFile
+  alias ElixirSearchExtractor.SearchKeyword.Schemas.Keyword
+
+  def user_keywords(user) do
+    from keyword in Keyword,
+      join: file in KeywordFile,
+      on: keyword.keyword_file_id == file.id,
+      where: file.user_id == ^user.id,
+      order_by: fragment("? DESC", keyword.inserted_at)
```

Comment:
I think `fragment` is not needed here to make the code simpler
```suggestion
      order_by: [desc: :inserted_at]
```